scripts/acn/run_acn_node_standalone.py

```diff
@@ -0,0 +1,387 @@
+        log_file = config.get(AcnNodeConfig.ACN_LOG_FILE, "")
+
+        return cls(
+            key,
+            uri,
+            external_uri,
+            delegate_uri,
+            monitoring_uri,
+            entry_peers.split(","),
+            log_file,
+            enable_checks,
+        )
+
+    @staticmethod
+    def check_config(config: Dict[str, str]) -> None:
+        """
+        Validate an ACN node configuration.
+
+        :param config: dictionary containing the configuration to check
+        """
+
+        SigningKey.from_string(
+            unhexlify(config[AcnNodeConfig.KEY]), curve=curves.SECP256k1
+        )
+
+        AcnNodeConfig._check_uri(config[AcnNodeConfig.URI])
+        if config[AcnNodeConfig.EXTERNAL_URI] != "":
+            AcnNodeConfig._check_uri(config[AcnNodeConfig.EXTERNAL_URI])
+        if config[AcnNodeConfig.DELEGATE_URI] != "":
+            AcnNodeConfig._check_uri(config[AcnNodeConfig.DELEGATE_URI])
+        if config[AcnNodeConfig.MONITORING_URI] != "":
+            AcnNodeConfig._check_uri(config[AcnNodeConfig.MONITORING_URI])
+
+        maddrs = config[AcnNodeConfig.ENTRY_PEERS_MADDRS].split(
+            AcnNodeConfig.LIST_SEPARATOR
+        )
+        for maddr in maddrs:
+            AcnNodeConfig._check_maddr(maddr)
+
+    @staticmethod
+    def _check_uri(uri: str) -> None:
+        """Check uri."""
+        if uri == "":
+            return
+        parts = uri.split(":")
+        if len(parts) != 2:
+            raise ValueError("Malformed uri '{}'".format(uri))
+        int(parts[1])
+
+    @staticmethod
+    def _check_maddr(maddr: str) -> None:
+        """Check multiaddress."""
+        if maddr == "":
+            return
+        parts = maddr.split("/")
+        if len(parts) != 7:
+            raise ValueError("Malformed multiaddress '{}'".format(maddr))
+        multihashdecode(b58decode(parts[-1]))
+
+
+class AcnNodeStandalone:
+    """Deploy an acn node in standalone mode."""
+
+    def __init__(self, config: AcnNodeConfig, libp2p_node_binary: str):
+        """
+        Initialize a new AcnNodeStandalone object.
+
+        :param config: node's configuration
+        :param libp2p_node_binary: path to libp2p node binary
+        """
+
+        self.config = config
+        self.binary = libp2p_node_binary
+        self._proc = None  # type: Optional[subprocess.Popen]
+
+    def run(self):
+        """Run the node."""
+        config_file = ".acn_config"
+        self.config.dump(config_file)
+
+        cmd = [self.binary, config_file]
+
+        if self.config.config[AcnNodeConfig.ACN_LOG_FILE] != "":
+            self._proc = subprocess.Popen(  # nosec
+                cmd,
+                shell=False,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.STDOUT,
+                bufsize=1,
+            )
+            with open(
+                self.config.config[AcnNodeConfig.ACN_LOG_FILE], "ab", 1
```

Comment:
not quite sure about this. Documentation states https://docs.python.org/3/library/functions.html#open

    buffering is an optional integer used to set the buffering policy.
    Pass 0 to switch buffering off (only allowed in binary mode), 1 to select
    line buffering (only usable in text mode), and an integer > 1 to indicate
    the size of a fixed-size chunk buffer.

Reply:
Lifted directly from fetch. I am not eager to go ahead and modify it.